install_files/ansible-base/roles/build-grsec-metapackage-deb-pkg/tasks/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+- include: build_grsec_metapackage_deb_pkg.yml
```

Comment:
Since this role `build-grsec-metapackage-deb-pkg` only builds the grsec-metapackage, should move the contents of ` install_files/ansible-base/roles/build-grsec-metapackage-deb-pkg/tasks/build_grsec_metapackage_deb_pkg.yml` to ` install_files/ansible-base/roles/build-grsec-metapackage-deb-pkg/tasks/main.yml` instead of including the separate task file.